src/main/java/com/github/junit5docker/Port.java

```diff
@@ -1,7 +1,21 @@
 package com.github.junit5docker;
 
+/**
+ * Describes a port binding to send to docker.
+ *
+ * @since 1.0
+ */
 public @interface Port {
+
+    /**
+     * @return the port number exposed on localhost.
```

Comment:
@FaustXVI hum, this is the tricky part, you don't know if it's localhost or anything really. It depends on the `docker` daemon you're talking to.